TwilioVerifySDK/TwilioVerify/Sources/Data/Storage.swift

```diff
@@ -86,20 +71,55 @@ extension Storage: StorageProvider {
   }
 }
 
+// MARK: - Storage Migrations Implementation
+
 private extension Storage {
+
+  // MARK: Properties
+
+  var isAppExtension: Bool {
+    return Bundle.main.bundlePath.hasSuffix(Constants.appExtensionSuffix)
+  }
+
+  var storedCurrentVersion: Int {
+    get { userDefaults.integer(forKey: Constants.currentVersionKey) }
+    set { userDefaults.set(version, forKey: Constants.currentVersionKey) }
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/6449b23696fdb9d5569ea5e2b08cc9943fdd6c51/TwilioVerifySDK/TwilioVerify/Sources/Data/Storage.swift#L86"></span>Setter value is not used.<br />
<code>unused_setter_value</code> <code>Storage.swift:86</code></td>
    </tr>
  </tbody>
</table>